catalogue_pipeline/transformer/src/test/scala/uk/ac/wellcome/platform/transformer/transformers/sierra/SierraConceptsTest.scala

```diff
@@ -81,25 +76,27 @@ class SierraConceptsTest extends FunSpec with Matchers {
     )
   }
 
-  it("puts extra instances of subfield 0 into the otherIdentifiers field") {
-    val caught = intercept[RuntimeException] {
-      transformer.identifyPrimaryConcept[Concept](
-        concept = Concept(label = "Hitchhiking horses hurry home"),
-        varField = VarField(
-          fieldTag = "p",
-          marcTag = "655",
-          indicator1 = "",
-          indicator2 = "0",
-          subfields = List(
-            MarcSubfield(tag = "a", content = "hitchhiking"),
-            MarcSubfield(tag = "0", content = "u/xxx"),
-            MarcSubfield(tag = "0", content = "u/yyy")
-          )
+  it("ignores multiple instances of subfield 0 im the otherIdentifiers") {
+    val concept = Concept(label = "Hitchhiking horses hurry home")
```

Comment:
typo: s/im/in

Reply:
I agree it's better to process these records than to throw errors, but I'm starting to wonder about the amount of data cleaning we are pushing into the transformers.